### PR TITLE
VUM based installation steps (fixes #555) [SKIP CI]

### LINKED ISSUE #555: Add documentation for VUM
Add documentation for VUM based installs.

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
VUM based installation steps (fixes #555)

Steps were performed while testing VIB installation (VUM compatible zip file) .. see #1198

/cc @govint @pdhamdhere

### CODE REVIEW

File: jekyll-docs/install.md [hunk 1]
```diff
@@ -3,10 +3,27 @@ title: Installation
...
+## Installation on ESXi
+
+[VIB](http://blogs.vmware.com/vsphere/2011/09/whats-in-a-vib.html) and [Offline Depot](https://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html?resultof=%2522%256f%2566%2566%256c%2569%256e%2565%2522%2520%2522%256f%2566%2566%256c%2569%256e%2522%2520%2522%2564%2565%2570%256f%2574%2522%2520) are the packages built to install the backend for the service on ESX. The backend can be installed using esxcli or vmware tools such as [VUM](http://pubs.vmware.com/vsphere-60/topic/com.vmware.ICbase/PDF/vsphere-update-manager-601-install-administration-guide.pdf)
+
+### VUM based VIB installation
+ESXi component for vDVS is available in the form of an [Offline Depot](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html) and available to download latest release as [ZIP archive](https://bintray.com/vmware/vDVS/VIB/_latestVersion).
+
+After downloading VIB bundle as zip format, installation is performed using VUM using following steps.
```
Comment (msterin) [inline jekyll-docs/install.md]: using the following steps

File: jekyll-docs/install.md [hunk 2]
```diff
@@ -3,10 +3,27 @@ title: Installation
...
+
+1. Go to admin view in update manager tab.
+2. Import zip bundle using import zip option.
```
Comment (msterin) [inline jekyll-docs/install.md]: nit - add  *italic* to *admin view* , *import zip*

File: jekyll-docs/install.md [hunk 3]
```diff
@@ -3,10 +3,27 @@ title: Installation
...
+3. Create a baseline and include zip bundle.
+4. Scan your host against baseline.
+5. Remediate host.
+
+More information can be found at [VUM User guide](https://featurewalkthrough.vmware.com/#!/vsphere-6-5/vsphere-update-manager-overview-cluster-upgrade/1).
+
+### VIB installation through esxcli/localcli
+
+ESXi component for vDVS is available in the form of a [VIB](https://blogs.vmware.com/vsphere/2011/09/whats-in-a-vib.html). VIB stands for vSphere Installation Bundle. At a conceptual level a VIB is somewhat similar to a tarball or ZIP archive in that it is a collection of files packaged into a single archive to facilitate distribution.
```
Comment (msterin) [inline jekyll-docs/install.md]: "somewhat" word is not needed

File: jekyll-docs/install.md (hunk 1 above)
Comment (shuklanirdesh82) [inline jekyll-docs/install.md]: done!

File: jekyll-docs/install.md (hunk 2 above)
Comment (shuklanirdesh82) [inline jekyll-docs/install.md]: done!

**Comment by shuklanirdesh82** [comment]
Thanks @msterin for the quick review, I have addressed your comments. Going to merge this documentation change.

**Pull request merged**